docs/main/releases.md

```diff
@@ -2,39 +2,106 @@
+
+- **Restore across Multiple Replicas**
+
+The capability to restore from snapshots across multiple replicas has been introduced in recent releases, enhancing data recovery options​.
+
+- **Expansion of Volumes with Snapshots**
+
+This release includes support for volume expansion even when snapshots are present.
+
+- **Placement of Replica Volumes across different Nodes/Pools**
+
+Replicated PV Mayastor now uses topology parameters defined in the storage class to determine the placement of volume replicas. This allows replicas to be controlled via labels from the storage class.
+
+- **Grafana Dashboards**
+
+Grafana Dashboards for Replicated PV Mayastor has been added in this releases.
+
+## Fixes
+
+### Fixed Issues - Local Storage
+
+- **Metrics Collection Loop**
+
+Adds an anonymous metrics collection loop which periodically pushes OpenEBS usage metrics. ([#188](https://github.com/openebs/dynamic-localpv-provisioner/pull/188),[#318](https://github.com/openebs/lvm-localpv/pull/318), and[#548](https://github.com/openebs/zfs-localpv/pull/548))
+
+### Fixed Issues - Replicated Storage
+
+- **Plugin changes for Snapshot Operation**
+
+This plugin will give detailed information about volume snapshot operation. ([#500](https://github.com/openebs/mayastor-extensions/pull/500))
+
+- **Deserialize Failures with Helm v3.13+ Installation**
+
+With Helm v3.13 or higher, helm chart values deserialize fails when loki-stack or jaeger-operator are disabled. This modification includes default deserialize options, which enable the essential options even when the dependent charts are disabled. ([#512](https://github.com/openebs/mayastor-extensions/pull/512))
+
+- **Scale of Volume**
+
+Earlier, the scale of volume was not allowed when the volume already has a snapshot. Now, Scale volume with snapshot can be used for replica rebuild. ([#826](https://github.com/openebs/mayastor-control-plane/pull/826))
+
+## Watch Items and Known Issues
+
+### Watch Items and Known Issues - Local Storage
+
+Local PV ZFS / Local PV LVM on a single worker node encounters issues after upgrading to the latest versions. The issue is specifically associated with the change of the controller manifest to a Deployment type, which results in the failure of new controller pods to join the Running state. The issue appears to be due to the affinity rules set in the old pod, which are not present in the new pods. As a result, since both the old and new pods have relevant labels, the scheduler cannot place the new pod on the same node, leading to scheduling failures when there's only a single node.
+The workaround is to delete the old pod so the new pod can get scheduled. See the issue [#3741](https://github.com/openebs/openebs/issues/3751) for more details.
+
+### Watch Items and Known Issues - Replicated Storage
+
+- Replicated PV Mayastor does not support the capacity expansion of DiskPools as of v2.6.0.
```

Comment:
v2.7.0

Reply:
We have to change this here as well Vishnu: https://github.com/openebs/openebs/releases/tag/v4.1.0